public/index.html

```diff
@@ -46,8 +55,13 @@
     <meta property="twitter:image" content="/thumbnail.png">
   </head>
   <body>
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KH3S5G4"
```

Comment:
@sama007 Don't we have GA already?
https://github.com/covid19india/covid19india-react/blob/8b979ecc855593289bc139cfd2e3c782cdfe6377/public/index.html#L77

Reply:
Yes. There was already a GA tag was being used. But I guess that was for the entire site. As the Essentials tab has a lot of user interaction(button clicking, accordion expanding, keyword searching etc.), we wanted to keep a separate track of this. So, separate GA tag was generated. This GA tracking used in this page some separate triggers and events to specifically monitor the ux of this page.

Reply:
@sama007 I think if the multiple tags are not configured correctly when they are included then it will start sending out incorrect data. Can you please confirm?